cmd/send.go

```diff
@@ -139,7 +124,7 @@ var sendCmd = &cobra.Command{
 
 func init() {
 	sendCmd.Flags().StringArrayVarP(&sendDataArgs, "data", "d", []string{}, "Set the data to send to specified host (prepend @ for files)")
-	sendCmd.Flags().StringVarP(&sendInputFormat, "format", "F", util.FORMAT_JSON, "Set the default data input format (json|yaml) can be overridden by file extension")
+	sendCmd.Flags().VarP(&sendInputFormat, "format", "F", "Set the default data input format (json|yaml) can be overridden by file extension")
```

Comment:
Did you want to add shell completion for this variable as well? (And for other commands taking `--format` as well.)

E.g:

https://github.com/OpenCHAMI/ochami/blob/caf429e9d1ab52f54269401c21a75c967fe62fb3/cmd/bss-dumpstate.go#L54

https://github.com/OpenCHAMI/ochami/blob/caf429e9d1ab52f54269401c21a75c967fe62fb3/cmd/lib.go#L513-L521

It doesn't look like Magellan generates shell completions in its releases so this may be irrelevant. But, something to consider if that is ever added.